.php_cs.dist

```diff
@@ -35,7 +35,6 @@ return PhpCsFixer\Config::create()
         'psr4' => false,
         'self_accessor' => false,
         'yoda_style' => null,
-        'non_printable_character' => true,
```

Comment:
Why?

Reply:
Because the pattern use non breakable spaces to display price (`@nbsp;`) I need this character in PatternTransformer to generate the appropriate pattern, but with this option cs-fixer converted all of them into regular spaces..
This worries me because someone could add it back later without knowing or realizing that it would change the behaviour Maybe an explicit false with a comment would be safer

Reply:
I didn't find a way to add the nbsp char in a string, even with `chr`, the extended char code :( 
You maybe need to update this too: https://github.com/PrestaShop/php-dev-tools/blob/master/src/CsFixer/Config.php#L42

Reply:
@PierreRambaud What about this:?
var_dump(json_encode(mb_convert_encoding('&nbsp;', 'UTF-8', 'HTML-ENTITIES')));
var_dump(json_encode("\xC2\xA0"));